client/src/components/Jobs/JobsStatus/JobsStatus.module.scss

```diff
@@ -1,21 +1,19 @@
-$font-weight: 500;
-
 .root {
+  /* Fix size of `.badge` and `.icon` */
   :first-child {
     font-size: 100%;
   }
 }
 
 .open-icon {
   margin-right: 0.25rem;
-  font-weight: $font-weight;
 }
 .open-button {
   margin-left: 0.5rem;
   min-height: 1.25rem;
   background-color: #f4f4f4;
   border: 1px solid #afafaf;
-  font-weight: $font-weight;
+  font-weight: 500;
```

Comment:
Is `500` desired here?

Reply:
**Yup.** The `.open-button` is the "↗ Open Session" button ([design](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/)). I'll add this change to the "Summary of Changes" for that file.

Reply:
**Update**: Added "retained 500 for text, but do _not_ use variable (_still_ match design)".